sdl_android/src/main/java/com/smartdevicelink/transport/USBAccessoryAttachmentActivity.java

```diff
@@ -64,20 +108,86 @@ private void checkUsbAccessoryIntent(String sourceAction) {
         Log.d(TAG, sourceAction + " with action: " + action);
 
         if (UsbManager.ACTION_USB_ACCESSORY_ATTACHED.equals(action)) {
-            Intent usbAccessoryAttachedIntent =
-                    new Intent(USBTransport.ACTION_USB_ACCESSORY_ATTACHED);
-            usbAccessoryAttachedIntent.putExtra(UsbManager.EXTRA_ACCESSORY,
-                    intent.getParcelableExtra(UsbManager.EXTRA_ACCESSORY));
-            usbAccessoryAttachedIntent
-                    .putExtra(UsbManager.EXTRA_PERMISSION_GRANTED,
-                            intent.getParcelableExtra(
-                                    UsbManager.EXTRA_PERMISSION_GRANTED));
+            Intent usbAccessoryAttachedIntent =  new Intent(USBTransport.ACTION_USB_ACCESSORY_ATTACHED);
+            usbAccessoryAttachedIntent.putExtra(UsbManager.EXTRA_ACCESSORY,intent.getParcelableExtra(UsbManager.EXTRA_ACCESSORY));

```

Comment:
we should probably wrap these in an `if api >= 12` since `UsbManager` is not available under 12 and our min is 8